### PR TITLE
Financroo user store

### PR DESCRIPTION
**Pull request by ikawalec**
- user repo based on file-based key-value storage: https://github.com/boltdb/bolt

### CODE REVIEW

**Review by mbilski — APPROVED** [review]

**Review by bwereszczak — APPROVED** [review]

File: financroo/db.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"time"
+)
+
+func InitDB() (*bolt.DB, error) {
+	var (
+		db  *bolt.DB
+		err error
+	)
+
+	if db, err = bolt.Open("./my.db", 0600, &bolt.Options{Timeout: 3 * time.Second}); err != nil {
```
Comment (holowinski) [inline financroo/db.go:15]: If we're not going to mount a volume this is going to disappear after docker restart - which is not that different from having no db at all.
Do we have a mount which would keep this on the host's disk ?

**Comment by holowinski** [comment]
> * user repo based on file-based key-value storage: https://github.com/boltdb/bolt

From what I can see this repo is obsolete / archived.
Could we switch to the recommended https://github.com/etcd-io/bbolt ?

**Comment by ikawalec** [comment]
> > * user repo based on file-based key-value storage: https://github.com/boltdb/bolt
> 
> From what I can see this repo is obsolete / archived.
> Could we switch to the recommended https://github.com/etcd-io/bbolt ?

Done

File: financroo/db.go (hunk 1 above)
Comment (ikawalec) [reply financroo/db.go:15]: Currently, this repo is not used. I'll add a mount in the next PR where the actual integration happens.

**Review by holowinski — APPROVED** [review]

**Pull request merged**